### PR TITLE
Support multiple extension by configuraion

### PR DESCRIPTION
**Pull request by nati**

### CODE REVIEW

File: extension/multienv.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+// implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package extension
+
+import "github.com/cloudwan/gohan/schema"
+
+//MutiEnvironment can handle multiple environment
+type MutiEnvironment struct {
```
Comment (morrisson) [inline extension/multienv.go]: s/Muti/Multi/

Comment (nati) [inline extension/multienv.go]: done. thx

**Comment by morrisson** [comment]
LGTM

**Comment by nati** [comment]
thx!

**Pull request merged**